main.go

```diff
@@ -83,6 +84,7 @@ var (
 	schemaOnly       bool
 	dataOnly         bool
 	sessionJSON      string
+	w                bool
```

Comment:
Probably use web instead of w.

While go loves to use single letter variables, these are typically only used for local variables in functions (and especially in loops) where you see their definition and use in the same page of code, and often only a few lines apart.

Anything that is a file-level variable or constant should generally have something more descriptive because you need to remember it across the entire file. Same comment for public interfaces.

Reply:
using webapi instead of web as there is a packaged named web imported